### ISSUE TITLE
A11y_SQLAzureDataStudio_ExportBacpac_ExportBacpacSettings_ScreenReader: The required combo-box is not announced as required.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9183]
[“Check out Accessibility Insights! -](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio; #A11yMAS; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #Win32; #MAS1.3.1; #JAWS; #NVDA; #MAS3.3.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: JAWS,NVDA
MAS References:MAS1.3.1, MAS3.3.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select forth radio button.
9. Navigate till next button & hit enter.
10. Launch NVDA and tab tot he first combo-box.

**Actual:**
On navigating to the first combo-box "source server" screen reader does not annouce it as required which is visually seen on screen. Simmilarly for "source database"

Same issue with JAWS

**Expected**:
When navigated to the combo-box with tab key the name of the combo-box the itemname as well as the instrcution as required must be announced.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen readers users will not know that field is required.

**MAS Reference:**
MAS 1.3.1:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS3.3.2:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EWLRFRV4A2ZOjDRjzOdNkOsBvUr5OQ1TGOw8UAT_BrfFRQ?e=e4WUKx

**Comment by kisantia** [comment]
The titles of the combo boxes are announced when navigating using the tab key
![comboBoxAnnouncing](https://user-images.githubusercontent.com/31145923/74782792-081e6c80-5259-11ea-831e-6275c0717c42.gif)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
